A3A/addons/core/functions/Supports/fn_initSupports.sqf

```diff
@@ -48,9 +48,9 @@ private _initData = [
     ["CARPETBOMBS",     "AREA", 0.5, 0.1, 200,   0, "u", ""],                            // balanced against airstrikes
     ["GUNSHIP",         "AREA", 0.2, 0.1, 0, 80, "", "vehiclesPlanesGunship"],                   //u      // uh. Does AREA work for this? Only lasts 5 minutes so maybe...
     ["SAM",           "TARGET", 1.0, 1.0,   0, 100, "u", ""],                            // balanced against ASF
-    ["CRUISEMISSILE", "AREA", 0.3, 0.1, 200,   100, "u", ""],
+    ["CRUISEMISSILE", "AREA", 0.4, 0.1, 150,   100, "u", ""],
```

Comment:
```suggestion
    ["CRUISEMISSILE", "AREA", 0.3, 0.1, 150,   100, "u", ""],
```
Don't really see any point of changing this, why does it need to be changed?

Reply:
I just think it should pop up just a tit bit more often, didn't see any reports about it

Reply:
If it uses the same laser object system as the UAV, it should probably be double checked to make sure it doesn't stack and kill the fps

Reply:
Such missiles are hugely expensive irl, its not something that would be deployed often, especially against weak guerrilla forces